_posts/2020-09-11-RestTemplate-vs-WebClient.md

```diff
@@ -295,25 +284,25 @@ public Flux<Recipe> getRecipes() {
 
 This way, you can still partly have a reactive setup and maybe when these services will be upgraded or replaced in the future, they might become reactive as well, and you can easily implement the necessary changes on your side as well.
 
-### Reactive Database Drivers
+### Reactive Database Connections
 
-There have been supporting libraries for Reactive NoSQL for a while, which can retrieve data from the database in a reactive manner. 
+There have been supporting libraries for reactive NoSQL connectivity for a while, which can retrieve data from the database in a reactive manner. 
 Which means they start returning data as the database is querying, and not when the entire database has been queried. So you are good to go. 
 But, with Relational databases (which you are probably using as well), there weren’t any in Spring until recently. 
 Before, you could wrap your database response in `Flux` and `Mono` wrappers as soon as possible when calling your database, but if these databases are the major sources of your data and not just a small portion, it kind of ruins the appeal and benefit. 
 
 ### R2DBC: 2 steps forward 1 step back
 
-Not until very recently (December 2019), `spring-data-r2dbc` dependency got released. 
-R2DBC stands for **Reactive Relational DataBase Connectivity**. Bear in mind that this driver is not an ORM. 
+Not until very recently (December 2019), `spring-data-r2dbc` dependency got released. It basically is JDBC's reactive counterpart. 
+R2DBC stands for **Reactive Relational DataBase Connectivity**. This does not support ORM (Object-Relational Mapping) at the moment of writing. 
```

Comment:
It does support ORM, but JPA doesn't work with it. There are othe ORM's than just JPA/hibernate. There is an ORM from spring itself: spring-data-r2dbc. It is very lightweight, but it's still an Object-Relational Mapping.

Reply:
True, you are correct. I've made some changes in the text.